test/functional/ex_cmds/mksession_spec.lua

```diff
@@ -0,0 +1,50 @@
+-- Test generation of session files
+
+local helpers = require('test.functional.helpers')(after_each)
+
+
+describe('A session file', function()
+  -- The file name use for each session
+  local session_file = 'Xtest-functional-ex_cmds-mksession.vim'
+
+  before_each(function()
+    helpers.clear()
```

Comment:
You should not use `helpers.…` anywhere, but just after `local helpers`: it is current convention that you “import” all entities you use via `local clear = helpers.clear\nlocal call = helpers.call\n…` (BTW, about `call`: `call('getcwd')` I normally express as `funcs.getcwd()`, it appears to also be more popular then `[helpers.]call`).